README.md

```diff
@@ -61,45 +61,112 @@ Your current working directory will now be mounted to the Unit image at `/www`.
 You can reach its socket at `/tmp/unit-control/control.unit.sock` assuming no
 further customizations have been made.
 
-### Amazon Linux, Fedora, Red Hat
+### Debian, Ubuntu, Amazon Linux, Fedora, Red Hat
 
+This helper script configures the correct package repositories for system.
 ``` console
 $ wget https://raw.githubusercontent.com/nginx/unit/master/tools/setup-unit && chmod +x setup-unit
-# ./setup-unit repo-config && yum install unit
-# ./setup-unit welcome
+# ./setup-unit repo-config
+```
+
+Debian derivatives:
+``` console
+# apt install unit
+```
+
+RHEL derivatives:
```

Comment:
Technically that's `Fedora derivatives` seeing as Fedora is the upstream for RHEL..